### ISSUE TITLE
Unselectable photos and empty albums when Optimized Storage is enabled

### ISSUE THREAD

**Issue by hanssens** [wymsee/cordova-imagePicker#57]
On iOS 8.2, when the feature "Optimized Storage" is enabled the idea is that all thumbnails are stored on the phone and fullres are in the cloud. With this feature enabled, in the ImagePicker, some of these photo's aren't even selectable. Some albums are even entirely empty.

This can be demonstrated on iOS 8.2 with the following sample code:

    window.imagePicker.getPictures(
        function (results) {
            for (var i = 0; i < results.length; i++) {
                console.log('Image URI: ' + results[i]);
            }
        }, function (error) {
            // no error occurs; it's a visual thing ;)
        }
    );

In short, there seems to be odd behaviour when using shared/streamed albums on iOS.

**Comment by CSullivan102** [comment]
Looks like this is likely the same root issue as https://github.com/wymsee/cordova-imagePicker/issues/56
